### DISCUSSION TITLE
Load custom models in Colab

### DISCUSSION THREAD

**Discussion by nikhilkofficial** [comfyanonymous/ComfyUI#195] [Q&A]
Hello, Thank you for this amazing UI. I'm trying to find a way to add my custom train models in the Colab version. Is there any possibility to do this as of now? It doesnt have an options to connect to GDrive.

**Comment by ltdrdata** [comment]
I have made a pull request regarding these features.

During that time, you can use this features on my implmentation.

GDrive mounting, refresh nodes for added models, ...

https://github.com/ltdrdata/ComfyUI
